examples/widgets/ellipse_selector_demo.py

```diff
@@ -0,0 +1,146 @@
+"""
+
+=====================
+Ellipse Selector Demo
+=====================
+
+Interactively selecting data in a region of interest in an image by drawing an
+ellipse.
+
+This example plots an image. You can then draw an ellipse shaped selection. To
+draw, click and drag to an extent within the image and then release.
+"""
+
+import numpy as np
+
+from matplotlib.patches import Ellipse
+from matplotlib.widgets import EllipseSelector
+
+
+class EllipseROIStats(object):
+    """
+    Select a region of an image using 'EllipseSelector' and extract the data.
+    The extents and center of the ellipse drawn are used to create a patch
+    in the same area as the selector object. The patch is the then used to
+    generate a mask that can be used to extract data.
+
+    Parameters
+    ----------
+    ax: `~matplotlib.axes.Axes`: Axes to interact with
+    img_data: `numpy.ndarray`: Image array
+    """
+
+    def __init__(self, ax, img_data):
+        self.ax = ax
+        self.fig = ax.figure
+
+        lineprop = dict(color='black', linestyle="-.", linewidth=2,
+                        dashes=(3, 1), alpha=1.0)
+        state_mods = dict(move=' ', clear='escape')
+        self.ellipse_roi = EllipseSelector(self.ax, onselect=self.on_select,
+                                           drawtype='line', lineprops=lineprop,
+                                           state_modifier_keys=state_mods,
+                                           interactive=True)
```

Comment:
You set `interactive=True` to add the rectangle handles. However the ellipse an it's handles are removed before one could interact with them. Either don't show the handles, or let the selector persist.